eks-deployments/instance-domain-join.md

```diff
@@ -45,7 +45,7 @@ aws ssm create-association --name $domainjoinSSMdoc --document-version 1 --targe
 aws ssm list-associations --association-filter-list "key=Name, value=$domainjoinSSMdoc"
 ```
 
-## 4. Create and join gMSA AD security group (Optional)
```

Comment:
Don't remove the keyword (Optional) because this step is not needed when you add more instances to the gMSA setup at a later stage.